### PR TITLE
SR-109: Add data-fusion-query to search templates

### PR DESCRIPTION
**Pull request by igkislev**

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
Please documents this change.

Add a version bullet here: https://github.com/wet-boew/GCWeb/blob/master/templates/search/api.html#L66

This is a minor template change.

Document the code addition in the section "Template - Search bar", "Template - Search bar contextual" and "Template -Alternate 'Search bar' "

Add the new code sample and archive the current one into a details/summary like the following

```html
<pre><code>
... The updated html here with data-fusion-query ...
</code></pre>
<details>
   <summary>as per version 2.1</summary>
<pre><code>
... The html that was already documented in the page ...
</code></pre>
</details>
```

File: templates/search/results-contextual-en.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <div class="well">
 	<form class="mrgn-tp-sm">
 		<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
-		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search">
+		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query="safe" />
```
Comment (duboisp) [inline templates/search/results-contextual-en.html:20]: @igkislev can you remove the value "safe"? That parameter value have no effect.
```suggestion
		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query />
```

File: templates/search/results-en.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 <form>
 	<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
 	<div class="input-group mrgn-tp-lg">
-		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" />
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query="safe"  name="q" />
```
Comment (duboisp) [inline templates/search/results-en.html:20]:
```suggestion
		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query  name="q" />
```

File: templates/search/results-fr.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
 <form>
 	<label class="wb-inv" for="sch-inp">Recherchez le site Web du Gouvernement du Canada</label>
 	<div class="input-group mrgn-tp-lg">
-		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" />
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query="safe" name="q" />
```
Comment (duboisp) [inline templates/search/results-fr.html:19]:
```suggestion
		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query name="q" />
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]
@igkislev 

* [ ] Please documents this change.
* [ ] Add a version bullet here: https://github.com/wet-boew/GCWeb/blob/master/templates/search/api.html#L66
* [ ] Document the code addition in the section "Template - Search bar", "Template - Search bar contextual" and "Template -Alternate 'Search bar' "
* [ ] Add the new code sample and archive the current code

File: templates/search/results-en.html [hunk 4]
```diff
@@ -17,7 +17,7 @@
 <form>
 	<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
 	<div class="input-group mrgn-tp-lg">
-		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" />
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query  name="q" />
```
Comment (GormFrank) [inline templates/search/results-en.html:20]: Suuuuper small request. Can you please remove one space just for the purpose of having consistent clean code? Not an error, I<m just really picky sometimes 😛 

```suggestion
		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query name="q" />
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Approved after the automated build is completed and after a quick local testing to see how the change made are displayed.

**Review by GormFrank — CHANGES_REQUESTED** [review]
Apart from those small changes requested, the rest looks good. Thank you @igkislev!

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — APPROVED** [review]

File: templates/search/results-contextual-fr.html [hunk 5]
```diff
@@ -38,7 +38,8 @@
 <section class="results">
 	<h2 class="wb-inv">Résultats de recherche</h2>
 	<p class="h4">[nombre de] résultats de recherché pour "[le terme recherché]"</p>
-
+	<p>De [Context label]</p>
+
```
[the remark this answers is not in the document]
Comment (GormFrank) [reply templates/search/results-contextual-fr.html:42]: @igkislev This one is still an issue, and it is the last one it seems. You need to remove the empty spaces / tab.

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**